pkgs/development/libraries/fftw/default.nix

```diff
@@ -25,6 +25,10 @@ stdenv.mkDerivation (finalAttrs: {
     sha256 = "sha256-VskyVJhSzdz6/as4ILAgDHdCZ1vpIXnlnmIVs0DiZGc=";
   };
 
+  patches = [
+    ./remove_missing_FFTW3LibraryDepends.patch
```

Comment:
There should likely be comments in here documenting:
1. Where the patch comes from (backported from upstream? custom?)
2. Why we need it

Depending on (2), this should possibly also be sent upstream. We usually much prefer backports of upstream accepted patches to custom patches that need to be maintained in nixpkgs long term.

Reply:
I a modified form of the patch upstream: https://github.com/FFTW/fftw3/pull/338
You can use `fetchpatch` to get it from there.

Reply:
thanks!